lib/Ilbot/Config.pm

```diff
@@ -79,12 +79,20 @@ sub config {
         $d = $d->{$_};
         $c //= $d;
         unless (defined $c) {
-            die "Can't find config for @keys (config missing from $_)";
+            die "Can't find config for @keys (config missing from $_)"
+                unless $_ eq 'chan_conf';
```

Comment:
I'm not happy about a special case for `chan_conf` here.

I think adding an entry (empty hash ref) to `%defaults` should also do the trick.